### PR TITLE
feat: show loader timing

### PR DESCRIPTION
**Pull request by popomore**
Ref https://github.com/eggjs/egg/issues/1898

##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

File: agent.js [hunk 1]
```diff
@@ -25,6 +28,14 @@ module.exports = agent => {
   // watch dirs to reload worker, will debounce 200ms
   agent.watcher.watch(watchDirs, debounce(reloadWorker, 200));
 
+  agent.beforeStart(function* () {
+    const rundir = agent.config.rundir;
+    const files = yield fs.readdir(rundir);
+    for (const file of files) {
+      yield rimraf(path.join(rundir, file));
```
Comment (atian25) [inline agent.js]: 只删除 time_pid 文件就好了？或者直接删除目录？

**Comment by atian25** [comment]
cov 挂了

File: agent.js (hunk 1 above)
Comment (popomore) [reply agent.js]: 嗯，直接删除好了，本地应该也没用，不然这个文件越来越多会很慢

**Comment by popomore** [comment]
Wait https://github.com/eggjs/egg/pull/2531

**Comment by atian25** [comment]
development mode only has one worker

**Comment by popomore** [comment]
> development mode only has one worker

What does that mean?

**Comment by popomore** [comment]
合并跑跑看

**Comment by popomore** [comment]
+ egg-development@1.4.0

**Pull request merged**